dffml/operation/db.py

```diff
@@ -91,6 +133,72 @@ async def db_query_create_table(
     ctx_enter={"dbctx": (lambda self: self.parent.database())},
 )
 async def db_query_insert(self, *, table_name: str, data: Dict[str, Any]):
+    """
+    Generates an insert query in the database.
+
+    Parameters
+    ++++++++++
+    table_name : str
+        The name of the table to insert data in to.
+    data : dict
+        Data to be inserted into the table.
+
+    Examples
+    ++++++++
+
+    >>> dataflow = DataFlow(
+    ...     operations={
+    ...         "db_query_insert": db_query_insert.op,
+    ...         "db_query_lookup": db_query_lookup.op,
+    ...         "get_single": GetSingle.imp.op,
+    ...     },
+    ...     configs={
+    ...         "db_query_lookup": DatabaseQueryConfig(database=sdb),
+    ...         "db_query_insert": DatabaseQueryConfig(database=sdb),
+    ...     },
+    ...     seed=[],
+    ... )
+    >>>
+    >>> inputs = {
+    ...     "insert":[
+    ...         Input(
+    ...             value="myTable", definition=db_query_insert.op.inputs["table_name"],
+    ...         ),
+    ...         Input(
+    ...            value={"key": 10, "firstName": "John", "lastName": "Doe", "age": 16},
+    ...             definition=db_query_insert.op.inputs["data"],
+    ...         ),
+    ...     ],
+    ... 
+    ...     "lookup": [
+    ...         Input(
+    ...             value="myTable", definition=db_query_lookup.op.inputs["table_name"],
+    ...         ),
+    ...         Input(
+    ...             value={
+    ...                 "key": "real",
+    ...                 "firstName": "text",
+    ...                 "lastName": "text",
+    ...                 "age": "real",
+    ...             },
+    ...             definition=db_query_lookup.op.inputs["cols"],
+    ...         ),
+    ...         Input(value=[], definition=db_query_lookup.op.inputs["conditions"],),
+    ...         Input(
+    ...             value=[db_query_lookup.op.outputs["lookups"].name],
+    ...             definition=GetSingle.op.inputs["spec"],
+    ...         ),
+    ...     ]
+    ... }
+    >>>
+    >>> async def main():
+    ...     async for ctx, result in MemoryOrchestrator.run(dataflow, inputs):
+    ...         if(result!={}):
```

Comment:
The empty brackets in the outputs made it hard to anticipate results (if we print all results, it prints empty dicts for those other than query lookups). Didn't wanna mess with the actual functions just for the sake of examples. I'm open to other suggestions, or if it's a green light I can figure something out inside the functions so they don't return those brackets. Was worried that might break other tests or something.